code/game/objects/items.dm

```diff
@@ -105,6 +105,10 @@ GLOBAL_VAR_INIT(rpg_loot_items, FALSE)
 
 	var/datum/rpg_loot/rpg_loot = null
 
+	var/in_inventory = FALSE//is this item equipped into an inventory slot or hand of a mob?
+	var/force_string = null//string form of an item's force
+	var/last_force_string_check = null

```

Comment:
This value is a number and should be declared as such. force_string is a string and should be declared as such.

You'll thank me for this if you ever use a real programming language.